AnnoyancesFilter/sections/subscriptions.txt

```diff
@@ -36,6 +36,9 @@ turkuazgazetesi.net,yenialanya.com,mansetaydin.com,seskocaeli.com,kocaelibarisga
 ~copays.org,~nothing.tech,~mega-debrid.eu,~curvegames.com,~news.un.org,~norwichzen.org.uk,~dereferer.me###mc_embed_signup
 !
 latest-ufo-sightings.net###text-3
+silicon.de###nl-overlay
+silicon.de##.wrap-contenus
+silicon.de##.register-newsletter
```

Comment:
`silicon.de##.register-newsletter` looks redundant due to `silicon.de##.wrap-contenus`